### PR TITLE
Add global RunAsMe configuration

### PR DESCRIPTION
**Pull request by fviale**
Similarly to the global fork mode, this setting allows to globally configure RunAsMe mode for all tasks submitted.

When this setting is true, it implies of course that global fork mode is also activated (other setting is ignored).

### CODE REVIEW

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJobFactory.java [hunk 1]
```diff
@@ -285,6 +284,10 @@ private static InternalTask createTask(Job userJob, JavaTask task) throws JobCre
                             JavaClassScriptEngineFactory.JAVA_CLASS_SCRIPT_ENGINE_NAME,
                             new Serializable[] { args }))));
                     javaTask.setForkEnvironment(task.getForkEnvironment());
+                    if (isRunAsMeTask()) {
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJobFactory.java]: MAJOR Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS134)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJobFactory.java [hunk 2]
```diff
@@ -329,6 +332,10 @@ private static InternalTask createTask(Job userJob, NativeTask task) throws JobC
             if (isForkingTask()) {
                 scriptTask = new InternalForkedScriptTask(new ScriptExecutableContainer(
                     new TaskScript(new SimpleScript(commandAndArguments, "native"))));
+                if (isRunAsMeTask()) {
```
Comment (lpellegr) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJobFactory.java]: The same code is duplicated 3 times, please factorize using a dedicated method.

**Comment by lpellegr** [comment]
:+1:

**Pull request merged**